test/entrypoint/srv_ddl.lua

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env tarantool
+
+require('strict').on()
+_G.is_initialized = function() return false end
+
+local log = require('log')
+local errors = require('errors')
+local cartridge = require('cartridge')
+local ddl = require('ddl')
+
+package.preload['customers-storage'] = function()
+    return {
+        role_name = 'customers-storage',
+        init = function()
+            local engine = os.getenv('ENGINE') or 'memtx'
+            local schema = {
+                spaces = {
+                    customers = {
+                        engine = engine,
+                        is_local = true,
+                        temporary = false,
+                        format = {
+                            {name = 'id', is_nullable = false, type = 'unsigned'},
+                            {name = 'bucket_id', is_nullable = false, type = 'unsigned'},
+                            {name = 'name', is_nullable = false, type = 'string'},
+                            {name = 'age', is_nullable = false, type = 'number'},
+                        },
+                        indexes = {{
+                            name = 'id',
+                            type = 'TREE',
+                            unique = true,
+                            parts = {
+                                {path = 'id', is_nullable = false, type = 'unsigned'},
+                            },
+                        }, {
+                            name = 'bucket_id',
+                            type = 'TREE',
+                            unique = false,
+                            parts = {
+                                {path = 'bucket_id', is_nullable = false, type = 'unsigned'},
+                            }
+                        }},
+                        sharding_key = {'id'},
```

Comment:
This schema has sharding key the same as primary key. This is too trivial. Test won't fail if I comment this line. 

Please add multipart primary key and choose several fields as sharding key. It's also will allow you to cover cases then key length is less than index length.

Reply:
Schema has been updated and fixed a test.